src/styles/blocks/card.css

```diff
@@ -0,0 +1,100 @@
+html {
+  font-family: "Roboto", sans-serif;
+
+  --card-border-color: #f3f3f3;
+  --text-color-black: #060b35;
+  --text-color-light: #616070;
+  --button-color: #00acdc;
+}
+
+.card {
+  padding: 32px 16px 16px;
+
+  width: 200px;
+  background: #fff;
+  border: 1px solid var(--card-border-color);
+  border-radius: 5px;
+}
+
+.card__photo {
+  width: 160px;
+  height: 134px;
+  display: block;
+  margin: 0 auto;
+}
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--text-color-black);
+
+  margin-top: 40px;
+}
+
+.card__product-code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--text-color-light);
+  
+  align-self: flex-start;
+  margin-top: 4px;
+}
+
+.card__stars {
+  display: flex;
+  margin-top: 16px;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.card__reviews {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--text-color-black);
+}
+
+.card__price {
+  margin-top: 24px;
+  display: flex;
+  justify-content: space-between;
+  line-height: 18px;
+}
+
+.card__price-world {
+  font-weight: 400;
+  font-size: 12px;
+  color: var(--text-color-light);
+}
+
+.card__price-number {
+  font-weight: 700;
+  font-size: 16px;
+  color: var(--text-color-black);
+}
+
+.card__link {
+  display: block;
+  margin-top: 16px;
+  padding: 12px;
+
+  text-decoration: none;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #fff;
+  
+  background: var(--button-color);
+  border: 1px solid transparent;
+  border-radius: 5px;
+}
+
+.card__link:hover {
+  color: var(--button-color);
+  background: #fff;
+  border-color: var(--button-color);
```

Comment:
you can move border directly to `.card__link` and don’t need to change it on hover